com.github.theCalcaholic.StreamDeck.yml

```diff
@@ -0,0 +1,77 @@
+app-id: com.github.theCalcaholic.StreamDeck
+runtime: org.freedesktop.Platform
+runtime-version: &runtime-version '22.08'
+sdk: org.freedesktop.Sdk
+command: streamdeck.sh
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=wayland
+  - --socket=fallback-x11
+  # Seems to be required for access to controllers/touch input
+  - --device=all
+  - --filesystem=/run/udev:ro
+  - --filesystem=xdg-data/Steam:rw
+  - --filesystem=~/.steam:rw
+  - --filesystem=~/.mozilla/firefox:rw
+  - --filesystem=~/.var/app/com.valvesoftware.Steam:rw
+  - --filesystem=~/.var/app/org.mozilla.firefox:rw
+  - --filesystem=xdg-data/flatpak:ro
+  - --talk-name=org.freedesktop.Flatpak
+  - --env=STREAMDECK_CMD_PREFIX=flatpak-spawn --host
+add-extensions:
+  org.freedesktop.Platform.ffmpeg-full:
+    directory: lib/ffmpeg
+    add-ld-path: .
+    version: *runtime-version
+modules:
+  - name: mtdev
+    buildsystem: autotools
+    sources:
+      - type: archive
+        url: https://bitmath.org/code/mtdev/mtdev-1.1.6.tar.gz
+        sha256: 1325f389a2f25cd5f5a8ea4d29aad24aa7c3ec30401d679400dd79eb9c0a8dbb
+      - type: shell
+        commands:
+          - cp -p /usr/share/automake-*/config.{sub,guess} config-aux
+  - name: xclip
+    buildsystem: autotools
+    sources:
+      - type: git
+        url: https://github.com/astrand/xclip.git
+        # Latest tag is from 2016, so tags probably aren't really used -> using latest master commit (at the time of submission)
+        commit: b372f73579d30f9ba998ffd0a73694e7abe2c313
+    modules:
+      - name: libx11-dev
+        buildsystem: autotools
+        sources:
+          - type: git
+            url: https://gitlab.freedesktop.org/xorg/lib/libx11.git
```

Comment:
I will reiterate. libX11 is in the runtime.

Reply:
Oh right. Sorry, I missed that